docker/Dockerfile.dmqnode

```diff
@@ -1,26 +1,62 @@
 # Duplicate definition of DIST before FROM is needed to be able to use it
 # in docker image name:
 ARG  DIST=xenial
-FROM sociomantictsunami/develdlang:$DIST-v6 as builder
+FROM sociomantictsunami/develdlang:$DIST-v7 as builder
 # Copies the whole project as makd needs git history:
 COPY . /project/
 WORKDIR /project/
 # Redefine arguments as env vars to be used inside build.sh script:
 ARG DIST=xenial
 ARG DMD=2.078.*
-ENV DMD=$DMD DIST=$DIST
-RUN docker/build.sh
+ENV DMD=$DMD
+RUN set -eu \
+    # Compiler configuration \
+    && . submodules/beaver/lib/dlang.sh \
```

Comment:
This is referring to beaver which I guess is not a submodule, otherwise you wouldn't have copy&pasted all this code?

Reply:
It's not a copy paste. This runs the beaver script for setting the `DMD` env variable. What looks like copied here sets the `PKG` variable for installing the right compiler package.

Reply:
But you say it is not (copied)?